### PR TITLE
DOCS-3849: Add custom URL in the Magento 2

### PR DESCRIPTION
**Pull request by Rendani-MultiSafepay**
Do the changes meet MultiSafepay Docs' Definition of Done?
- Style guide, tone of voice, and naming conventions
- Tested
- Reviewed on mobile, tablet and desktop screens
- Notified stakeholders
- Pass GitHub Actions

### CODE REVIEW

File: content/integrations/magento-2.md [hunk 1]
```diff
@@ -159,6 +159,21 @@ The base values are required if you have enabled the `use base currency` setting
 
 </details>
 
+## Custom urls
+
+<details id="how-to-set-up-custom-urls">
+<summary>How to set up custom URL</summary>
```
Comment (Vinod-MultiSafepay) [inline content/integrations/magento-2.md:165]: There are 2 custom urls to be set, the return url and the cancel url. Could you make this plural?

File: content/integrations/magento-2.md [hunk 2]
```diff
@@ -159,6 +159,21 @@ The base values are required if you have enabled the `use base currency` setting
...
+<br>
+
+1. Sign in to your Magento 2 backend.
+2. Go to **Stores** > **Configuration** > **MultiSafepay**.
+3. Select **General settings** > **Advanced settings**.
+4. In the **Use custom return urls** field, select **Yes**.
+5. In the **Custom redirect url after  canceling the payment** field, add your custom URL.
```
Comment (Vinod-MultiSafepay) [inline content/integrations/magento-2.md:172]: There are 2 custom urls, One says: `Custom redirect url after cancelling the payment`, the other one says: `Custom "Success page" url` , either one or both can be set. Please clarify that some more.

**Review by Vinod-MultiSafepay — CHANGES_REQUESTED** [review]
Good addition. Some small improvements can be made.

**Review by Vinod-MultiSafepay — APPROVED** [review]

File: content/integrations/magento-2.md [hunk 3]
```diff
@@ -157,6 +157,24 @@ If you have included a custom total and it is **not** being read, make sure it i
...
+<details id="how-to-set-up-custom-urls">
+<summary>How to set up custom URLs</summary>
+<br>
+
+1. Sign in to your Magento 2 backend.
+2. Go to **Stores** > **Configuration** > **MultiSafepay**.
+3. Select **General settings** > **Advanced settings**.
+4. In the **Use custom return urls** field, select **Yes**.
+5. Add your custom URL to either:
+  - **Custom redirect url after  canceling the payment** field, **or**
```
Comment (inesmayerMSP) [inline content/integrations/magento-2.md:173]: delete space

File: content/integrations/magento-2.md [hunk 4]
```diff
@@ -157,6 +157,24 @@ If you have included a custom total and it is **not** being read, make sure it i
...
+  - **Custom "Success page" url** field.
```
Comment (inesmayerMSP) [inline content/integrations/magento-2.md:174]: add your custom URL either to 
- the ....field or the ...field?

**Review by inesmayerMSP — CHANGES_REQUESTED** [review]

**Review by rossthomasmsp — APPROVED** [review]

File: content/integrations/magento-2.md (hunk 3 above)
Comment (Daniel-MultiSafepay) [reply content/integrations/magento-2.md]: This online MD editor suggest one empty space will do the trick to add indent to the bold line. 

https://markdownlivepreview.com/

Comment (Rendani-MultiSafepay) [reply content/integrations/magento-2.md]: @Daniel-MultiSafepay, delete space refers to the space between the text on line 174 between **after  canceling** there was double space.

**Pull request merged**